packages/test-interface-criteria/src/criteria/TestOpportunityBookableOutsideValidFromBeforeStartDate.js

```diff
@@ -1,34 +1,40 @@
-const moment = require('moment');
-
-const { TestOpportunityBookable } = require('./TestOpportunityBookable');
-const { createCriteria } = require('./criteriaUtils');
+const { InternalCriteriaFutureScheduledOpportunity } = require('./internal/InternalCriteriaFutureScheduledOpportunity');
+const { getRemainingCapacity, createCriteria, mustBeWithinBookingWindow } = require('./criteriaUtils');
 
 /**
+ * @typedef {import('../types/Criteria').OpportunityConstraint} OpportunityConstraint
  * @typedef {import('../types/Criteria').OfferConstraint} OfferConstraint
  */
 
 /**
  * @type {OfferConstraint}
  */
-function outsideValidFromBeforeStartDate(offer, opportunity) {
-  return (Array.isArray(offer.availableChannel) && offer.availableChannel.includes('https://openactive.io/OpenBookingPrepayment'))
-    && offer.advanceBooking !== 'https://openactive.io/Unavailable'
-    && (offer.validFromBeforeStartDate && moment(opportunity.startDate).subtract(moment.duration(offer.validFromBeforeStartDate)).isAfter());
+function mustBeOutsideBookingWindow(offer, opportunity) {
+  return offer.validFromBeforeStartDate && !mustBeWithinBookingWindow(offer, opportunity);
 }
 
 /**
- * Implements https://openactive.io/test-interface#TestOpportunityBookableOutsideValidFromBeforeStartDate
+ * @type {OpportunityConstraint}
  */
+function remainingCapacityMustBeAtLeastTwo(opportunity) {
+  return getRemainingCapacity(opportunity) > 1;
+}
```

Comment:
You could use the existing `remainingCapacityMustBeAtLeastTwo` function (from InternalTestOpportunityBookable.js) instead of creating a new one

Reply:
Actually, I would highly recommend that as Nick is updating that function in his ongoing PR. This way, they'll both be updated